SPACING.md

```diff
@@ -0,0 +1,335 @@
+# Spacing Style Guide
+When and where to use spaces and blank lines.
+
+### Rational
+(skip if bored)
+
+Humans primarily process and _make sense_ of data when consumed visually. Consistent spacing - and format in general - drastically lowers the cognitive load when reading code and decreases the time needed to understand it. We should consent to a single fixed style of formatting Swift files as part of a journey to better and more readable code. Hence, please enforce these guidelines when reviewing pull requests.
+
+__Note:__ All of the following suggestions are general guidelines. They should be subject to discussions and may be ignored if a convincing reason exists.
+
+---
+
+### Closing Scopes
+Simply __never__ include any whitespace before closing a scope and always __one__ blank line afterwards.
+```swift
+        return x
+    }
+}
+
+func nextFunc() {}
+```
+
+### Classes & Extensions
+Always use exactly __one__ empty line above and below classes and between every one of its methods.
+
+```swift
+//
+//  ViewController.swift
+//  Swift Styleguide
+//
+//  Created by Developer on 10.11.17.
+//  Copyright © 2017 intive_Kupferwerk. All rights reserved.
+//
+
+class ViewController {
+
+    // properties
+
+    func lastMethodInThisScope() {
+        // code
+    }
+}
+
+```
+
+### Properties
+
+- cluster variables in groups (static, class, public, internal, private, fileprivate, computed, observed, etc.) and separate them with __one__ blank line
+- always surround properties that have closures (observers, lazy, etc.) or documentation with __one__ blank line
+
+```swift
+struct SomeResource: Resource {
+
+    static let path = "/resource"
+
+    var internalPropertyOne: String
+    var internalPropertyTwo: String
+
```

Comment:
I don't like this example. IMHO we should show as a rule the properties that are documented, and then mention that _maybe_ sometimes it is _acceptable_ to have properties without documentation (and those properties then should e grouped in one block).